src/v3/src/types/schema.ts

```diff
@@ -516,6 +516,23 @@ export interface ImageWithTextElement extends UISchemaElement {
   };
 }
 
+export interface ImageLinkElement extends UISchemaElement {
+  type: 'ImageLink';
+  options: {
+    id: string;
+    href: string;
+    altText: string;
+    marginBlockStart: string;
```

Comment:
```suggestion
    marginBlockStart?: string;
```
Don't think we will always need to set a `marginBlockStart` for this component, let's make it optional. You'll have to update the `ImageLink` component to cover the case where a `marginBlockStart` isn't passed in as a prop anymore and set the margin to be 0 in that case

Reply:
please see https://github.com/okta/okta-signin-widget/commit/72f53833cd296a839665bc908434c845854c8679